server/lib/pubsubNotifiers/pubsubNotifier.js

```diff
@@ -0,0 +1,25 @@
+const notifiers = require('./notifiers')
+
+module.exports = function NewPubSub (pubsubConfig) {
+  const PubSubClass = notifiers[pubsubConfig.type]
+
+  if (!PubSubClass) {
+    throw new Error(`Unhandled pubsub type: ${pubsubConfig.type}`)
+  }
+
+  if (typeof PubSubClass !== 'function') {
+    throw new Error(`PubSub implementation for ${pubsubConfig.type} is missing a constructor`)
+  }
+
+  const pubsub = new PubSubClass(pubsubConfig.config)
```

Comment:
Is not better add all const on top of it? All together.

Reply:
@camilamacedo86 not sure what you mean, it wouldn't be possible to declare this at an earlier point in the code.

Reply:
I think it should be inside of the func getPublish (PubSubClass); as described in the next comment.